### ISSUE TITLE
1.4.11 - *active* user interface components?

### ISSUE THREAD

**Issue by jared-w-smith** [w3c/wcag21#617]
One phrase in this SC that is especially concerning to me is "**active** user interface components". The "active" state of components is defined as the state when the user is activating the control (see https://www.w3.org/wiki/CSS/Selectors/pseudo-classes/:active or search for "active" in the ARIA spec). This would very likely be interpreted to only require sufficient contrast during this state. I think the intent is to exempt inactive/disabled components, but "active" is not the opposite of "disabled" or "inactive" in this context.

I believe "active" should be removed from the SC language, and the definition for "user interface component" amended to exempt disabled components.

**Comment by carmacleod** [comment]
Actually, I guess the definition for "user interface component" needs to stay as-is, because it is already used in many other places that still need it to include inactive UI components. For example, an inactive component still needs to have name, role, value. 

So, if the intent of the word "active" in the Graphics Contrast SC is to exempt inactive/disabled components, then the language should specifically state that inactive/disabled components are exempt. This is what [Success Criterion 1.4.3 Contrast (Minimum)](https://w3c.github.io/wcag21/guidelines/#contrast-minimum) and [Success Criterion 1.4.6 Contrast (Enhanced)](https://w3c.github.io/wcag21/guidelines/#contrast-enhanced) both do.
This would allow us to remove the misleading/confusing word "active" from the phrase "~~active~~ user interface components" in [Success Criterion 1.4.11 Graphics Contrast](https://w3c.github.io/wcag21/guidelines/#graphics-contrast).

**Comment by allanj-uaag** [comment]
Possible solutions: 
find an antonym for ACTIVE" - perhaps non-disabled or inactive.  Now we are defining the component by what it is not. That promotes confusion.  

find an alternative word for ACTIVE - perhaps functioning or operating or operable. Hmm, if a component is non-disabled but broken (it does not do what it is supposed to do) is it still functioning or operating. That leaves Operable - able to be used. This seems to work. 

Proposal
User Interface Components
Visual information used to indicate states and boundaries of **operable** user interface components, except where the appearance of the component is determined by the user agent and not modified by the author;


@alastc @goodwitch

**Comment by alastc** [comment]
I think making it 'operable' user interface components works, and it causes the least change. It would get quite wordy with a full exception for inactive components.

**Comment by awkawk** [comment]
I don't think that it would be too wordy, and introducing a new term (new except in that we use it as the name for an entire principle).  To include mention of inactive would be like this:

Visual information used to indicate states and boundaries of user interface components, except for inactive components or where the appearance of the component is determined by the user agent and not modified by the author;

**Comment by jared-w-smith** [comment]
I fully support Andrew's wording. And I agree that this would be better than an addition of a new term/definition.

**Comment by allanj-uaag** [comment]
Andrew's wording works for me.

**Comment by alastc** [comment]
ok, that works.

**Comment by awkawk** [comment]
Accepted change in CFC a couple weeks ago. The change to my suggestion above is editorial from what was in the CFC.

**Comment by awkawk** [comment]
http://rawgit.com/w3c/wcag21/master/guidelines/index.html#graphics-contrast
